void tomo::IoLoop::run_loop<true, false, false, false, false>()
--- PRE

+++ POST

@@ -624,7 +624,7 @@

 0b8c [ 4] nopl 0x0(%rax)
 0b90 [ 5] mov $0x1,%esi
 0b95 [ 3] mov %rbx,%rdi
-0b98 [ 5] call <tomo::IoLoop::arm_accept(tomo::UrKind)>
+0b98 [ 5] call <tomo::IoLoop::fliplisten_retry_accept(tomo::UrKind)>
 0b9d [ 5] jmp <+0x2c2>
 0ba2 [11] data16 cs nopw 0x0(%rax,%rax,1)
 0bad [ 3] nopl (%rax)
